packages/botonic-react/src/index.d.ts

```diff
@@ -86,4 +95,108 @@ export function msgsToBotonic(
   customMessageTypes?: CustomMessageType[]
 ): React.ReactNode
 
+export interface WebchatAppArgs {
+  theme?: ThemeProps
+  persistentMenu?: PersistentMenuProps
+  coverComponent?: CoverComponentOptions
+  blockInputs?: BlockInputOption[]
+  enableEmojiPicker?: boolean
+  enableAttachments?: boolean
+  enableUserInput?: boolean
+  enableAnimations?: boolean
+  defaultDelay?: number
+  defaultTyping?: number
+  storage?: Storage
+  onInit: onOpen
+  onClose
+  onMessage
+  appId?: string
+  visibility?: () => boolean
+}
+
+export interface WebchatMessage {
+  id: string
+  type: core.InputType
+  data: any
+  timestamp: string
+  markdown: boolean
+  from: 'user' | 'bot'
+  buttons: ButtonProps[]
+  delay: number
+  typing: number
+  replies: ReplyProps[]
+  display: boolean
+  ack: 0 | 1
+}
+
+export interface OnUserInputArgs {
+  user: core.SessionUser
+  input: core.Input
+  // Below for WebchatDevApp
+  session?: core.Session
+  lastRoutePath?: string
+}
+
+export interface OnStateChangeArgs {
+  user: core.SessionUser
+  messagesJSON: WebchatMessage[]
+}
+
+export type MessageInfo = {
```

Comment:
why is MessageInfo a type and OnStateChangeArgs an interface?
I think it's almost the same, but we should be consistent